<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="0"
            failures="1"
            tests="6">

  <testsuite name="profile"
    errors="0" failures="1" hostname="" id="" package="profile" skipped="0"
    tests="6" time="143.4" timestamp="Tue, 20 Sep 2016 11:09:00 GMT">
  
    <testcase name="login" time="39.41" assertions="3">

    

    

    
    </testcase>
  
    <testcase name="Get User Name" time="22.00" assertions="13">

    

    

    

    

    

    

    

    

    

    

    

    

    
    </testcase>
  
    <testcase name="patient login" time="15.57" assertions="2">

    

    
    </testcase>
  
    <testcase name="Profile" time="46.30" assertions="15">

    

    

    

    

    

    

    

    

    

    

    

    

    

    

    
    </testcase>
  
    <testcase name="Admin Login" time="18.99" assertions="4">

    

    

    

    
    </testcase>
  
    <testcase name="Check" time="1.114" assertions="12">

    

    

    

    

    

      <failure message="Testing if value of &lt;#contact_no&gt; contains: &quot;9440498957&quot;.">    at Object.module.exports.Check (c:\dev1\nightwatch\node_modules\nightwatch\EHRuser\profile.js:278:14)
    at Module.call (C:\dev1\nightwatch\node_modules\nightwatch\lib\runner\module.js:63:34)
    at C:\dev1\nightwatch\node_modules\nightwatch\lib\runner\testcase.js:97:29
    at _fulfilled (C:\dev1\nightwatch\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\dev1\nightwatch\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\dev1\nightwatch\node_modules\q\q.js:796:13)
    at C:\dev1\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\dev1\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\dev1\nightwatch\node_modules\q\q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)</failure>

      <failure message="Testing if value of &lt;#homePhone&gt; contains: &quot;9356376877&quot;.">    at Object.module.exports.Check (c:\dev1\nightwatch\node_modules\nightwatch\EHRuser\profile.js:279:14)
    at Module.call (C:\dev1\nightwatch\node_modules\nightwatch\lib\runner\module.js:63:34)
    at C:\dev1\nightwatch\node_modules\nightwatch\lib\runner\testcase.js:97:29
    at _fulfilled (C:\dev1\nightwatch\node_modules\q\q.js:834:54)
    at self.promiseDispatch.done (C:\dev1\nightwatch\node_modules\q\q.js:863:30)
    at Promise.promise.promiseDispatch (C:\dev1\nightwatch\node_modules\q\q.js:796:13)
    at C:\dev1\nightwatch\node_modules\q\q.js:556:49
    at runSingle (C:\dev1\nightwatch\node_modules\q\q.js:137:13)
    at flush (C:\dev1\nightwatch\node_modules\q\q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)</failure>

    

    

    

    

    
    </testcase>
  
  
  </testsuite>
</testsuites>
